frame/bridge/ethereum/backing/src/lib.rs

```diff
@@ -165,10 +172,11 @@ decl_storage! {
 	trait Store for Module<T: Trait> as DarwiniaEthereumBacking {
 		pub VerifiedProof
 			get(fn verified_proof)
-			: map hasher(blake2_128_concat) EthereumTransactionIndex => Option<bool>;
+			: map hasher(blake2_128_concat) EthereumTransactionIndex => bool = false;
```

Comment:
Looks good, just to double check again:

Removing Option will not affect storage.

I remember last time @AsceticBear also asked the same question.

Reply:
I'm pretty sure. It won't affect the storage.

Reply:
It would be more convincing if more analysis about the storage influence provided or local test.